File: web/src/app/poto/class_order.ts
/**
 * 课程订单
 */
export class ClassOrder {
    constructor(
        public id?: number,
        public pId?: number,
        public pName?: string,
        public claId?: number,
        public claKName?: string,
        public uId?: number,
        public uName?: string,
        public cardId?: number,
        public cardKName?: string,
        public ordTime?: string,
        public num?: string
    ) {}

    static fromJSON(json: any): ClassOrder {
        const classOrder = Object.create(ClassOrder.prototype);
        Object.assign(classOrder, json);
        return classOrder;
    }

    toString() {
        return `
        ClassOrder {
            id: ${this.id},
            pId: ${this.pId},
            pName: ${this.pName},
            claId: ${this.claId},
            claKName: ${this.claKName},
            uId: ${this.uId},
            uName: ${this.uName},
            cardId: ${this.cardId},
            cardKName: ${this.cardKName},
            ordTime: ${this.ordTime},
            num: ${this.num}
        }`;
    }
}

File: app/app/src/main/java/bean/ClassOrder.java
package bean;

import java.util.Date;

/**
 * 课程预约信息
 * Created by Thor on 2018/4/5.
 */

public class ClassOrder {
    private Integer id;

    private Integer pId;

    private String pName;

    private Integer claId;

    private String claKName;

    private Integer uId;

    private String uName;

    private Integer cardId;

    private String cardKName;

    private Date ordTime;

    private Integer num;

    private Integer expend;

    private String property;

    private Integer isScore;

    public Integer getIsScore() {
        return isScore;
    }

    public void setIsScore(Integer isScore) {
        this.isScore = isScore;
    }

    public String getProperty() {
        return property;
    }

    public void setProperty(String property) {
        this.property = property;
    }

    public Integer getExpend() {
        return expend;
    }

    public void setExpend(Integer expend) {
        this.expend = expend;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getpId() {
        return pId;
    }

    public void setpId(Integer pId) {
        this.pId = pId;
    }


    public Integer getNum() {
        return num;
    }

    public void setNum(Integer num) {
        this.num = num;
    }

    public String getpName() {
        return pName;
    }

    public void setpName(String pName) {
        this.pName = pName;
    }

    public Integer getClaId() {
        return claId;
    }

    public void setClaId(Integer claId) {
        this.claId = claId;
    }

    public String getClaKName() {
        return claKName;
    }

    public void setClaKName(String claKName) {
        this.claKName = claKName;
    }

    public Integer getuId() {
        return uId;
    }

    public void setuId(Integer uId) {
        this.uId = uId;
    }

    public String getuName() {
        return uName;
    }

    public void setuName(String uName) {
        this.uName = uName;
    }

    public Integer getCardId() {
        return cardId;
    }

    public void setCardId(Integer cardId) {
        this.cardId = cardId;
    }

    public String getCardKName() {
        return cardKName;
    }

    public void setCardKName(String cardKName) {
        this.cardKName = cardKName;
    }

    public Date getOrdTime() {
        return ordTime;
    }

    public void setOrdTime(Date ordTime) {
        this.ordTime = ordTime;
    }
}
